youtube_dl/extractor/vier.py

```diff
@@ -49,13 +49,12 @@ def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         embed_id = mobj.group('embed_id')
         display_id = mobj.group('display_id') or embed_id
+        video_id = mobj.group('id') or embed_id
         site = mobj.group('site')
 
-        webpage = self._download_webpage(url, display_id)
-
-        video_id = self._search_regex(
-            [r'data-nid="(\d+)"', r'"nid"\s*:\s*"(\d+)"'],
-            webpage, 'video id')
+        webpage = self._download_webpage(
+            "http://www.%s.be/video/v3/embed/%s" % (site, video_id),
```

Comment:
Use single quotes. This change breaks all tests. No purpose of this change described, no tests. Change does not match PRs title.